__tests__/index.test.js

```diff
@@ -121,3 +121,61 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status 200: responds with an array of comment objects", () => {
+      return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+              expect(body[0].article_id).toBe(1);
+          body.forEach((comment) => {
```

Comment:
You should always check the length of the array before doing a foreach in a test (foreachs can produce false positives :))